src/components/calendar/EventCard.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+
+const EventCard = ({
+  name,
+  date,
+  time,
+  location,
+  logo,
+}: {
+  name: string;
+  date: string;
+  time: string;
+  location: string;
+  logo: string;
+}) => {
+  return (
+    <div className="flex w-full flex-col border-2 border-white">
+      <div className="flex justify-center bg-sw-gold-100 bg-gradient-to-r from-sw-gold-100 via-sw-white to-sw-gold-200">
+        <span className="my-[5%] text-3xl font-semibold text-white">
+          {name}
+        </span>
+      </div>
+      <div className="flex flex-col justify-center text-center">
+        <span className="text-3xl font-semibold text-white">{date}</span>
+        <span className="text-3xl font-semibold text-white">{time}</span>
+        <span className="text-3xl font-semibold text-white">{location}</span>
+        <Image src={logo} alt="Logo" className="absolute pt-20" />
```

Comment:
try to move the logo on top of the border line with -bottom-2 or [5%] then add a black bg and increase the z-index so it is on top of the border so we can replicate whats on the figma. 

if this doesn't work then leave as is

Reply:
The bottom class keeps moving the images to the very bottom so I used margins instead. let me know if there's better alternatives